paperqa/settings.py

```diff
@@ -932,6 +932,12 @@ async def make_ldp_agent(
             )
         raise NotImplementedError(f"Didn't yet handle agent type {agent_type}.")
 
+    def adapt_tools(self, tools):
+        if "gemini" in self.agent.agent_llm.lower():
```

Comment:
Can you link the LiteLLM issue https://github.com/BerriAI/litellm/issues/7634 , because in the future we can remove this logic if LiteLLM supports

Reply:
I stole your comment from aviary, is that ok?

Reply:
Yeah sure it's fine